src/backend/crew_ai/cleaned_llm_wrapper.py

```
@@ -45,6 +108,56 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         logger.info("🧹 Initialized CleanedLLMWrapper - will clean Action/ActionInput lines")
     
+    def call(self, messages, *args, **kwargs):
```

Ruff (0.15.6)

[warning] Missing type annotation for `*args`

(ANN002)